obspy/signal/src/evalresp/parse_fctns.c

```diff
@@ -19,7 +19,7 @@
 #include <stdlib.h>
 
 #include "./evresp.h"
-double atof();
```

Comment:
Is this not in `stdlib.h`? Can we not just delete the prototype?

Reply:
no idea... can't test myself on my machines

Reply:
confirmed, adapting & pushing